src/index.jsx

```diff
@@ -1,23 +1,29 @@
 /* eslint-disable react/wrap-multilines */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@babel/polyfill';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
+import ReduxToastr from 'react-redux-toastr';
 import Routes from './routes';
 import store from './store';
 
-
-
 function App() {
   return (
-    <div>
+    <Provider store={store}>
       <Routes />
-    </div>
+      <ReduxToastr
```

Comment:
#### Nitpick
- Why ?

Reply:
The plugin was initially used for demonstration purpose. However, I will make an update of a recommended toaster with support for easy customising: [react-toastify](https://github.com/fkhadra/react-toastify)